domain.yml

```diff
@@ -14,12 +14,15 @@ entities:
 slots:
   cuisine:
     type: text
-  location:
+  latitude:
+    type: text
+  location_name:
+    type: text
+  longitude:
     type: text
 templates:
   utter_greet:
-  - text: Hey!! I can help you to find restaurants based your preferred location and
-      cuisine.
```

Comment:
Keep response: Hey!! I can help you to find restaurants based on your preferred location and cuisine.